docs/library/network.WLAN.rst

```diff
@@ -85,7 +85,17 @@ Methods
         * ``STAT_GOT_IP`` -- connection successful.
 
     When called with one argument *param* should be a string naming the status
-    parameter to retrieve.  Supported parameters in WiFI STA mode are: ``'rssi'``.
+    parameter to retrieve, and different parameters are supported depending on the
+    mode the WiFi is in.
+
+    In STA mode, passing ``'rssi'`` returns a signal strength indicator value, whose
+    format varies depending on the port (currently supported on ESP32 and ESP8266).
+
+    In AP mode, passing ``'stations'`` returns a list of connected WiFi stations
+    (currently supported on ports/boards that use the CYW43 WiFi driver, and the
+    ESP32 and ESP8266 ports).  The format of the station information entries varies
```

Comment:
with this PR, stations is supported the same as rssi, on everything except cc3200